README.md

```diff
@@ -82,6 +44,80 @@ To find all duplicate files in all repositories run
 
 All repositories must be [hashed](#hash) before.
 
+## Requirements
+
+### Docker
+
+Start containers:
+
+```bash
+docker-compose up -d
+```
+
+Local directories `repositories` and `query` are available as `/repositories` and `/query` inside the container.
+
+Examples:
+
+```bash
+$ docker-compose exec gemini ./hash /repositories
+Hashing 2 repositories in: /repositories
+	file:/repositories/f281ab6f2e0e38dcc3af05360667d8f530c00103.siva
+	file:/repositories/9279be3cf07fb3cca4fc964b27acea57e0af461b.siva
+Done
+
+$ docker-compose exec gemini ./query /query/consumer.go
```

Comment:
Question: would that be possible to use single `docker-compose` just for running all the dependencies together and then keep examples here with just a `docker run`?

Asking, as I think from the product standpoint that may be conceptually a bit simple for some of the users.

Or do you think it would have some technical downsides?

Reply:
do you mean replace `docker-compose exec` with `docker exec`?
It wouldn't be simpler because the name of the container is something like `gemini_gemini_1`. And the last number can be different.